src/compiler/watchPublic.ts

```diff
@@ -118,8 +118,9 @@ export function readBuilderProgram(compilerOptions: CompilerOptions, host: ReadB
     return createBuilderProgramUsingProgramBuildInfo(buildInfo, buildInfoPath, host);
 }
 
-export function createIncrementalCompilerHost(options: CompilerOptions, system = sys): CompilerHost {
-    const host = createCompilerHostWorker(options, /*setParentNodes*/ undefined, system);
+// Dont think we need this since user can override getSourceFile on CompilerHost returned?

```

Comment:
This comment came out of the cherry pick; should I remove it or is there some detail I need to address?

Reply:
We can remove this, 
I forgot to thread through this from performIncrementalCompilation where it should be passed in as true

Reply:
https://github.com/microsoft/TypeScript/pull/52921/files#diff-5bc64df2f0c1746eae137760ddd218696b382812e3b408d417bd838f9620a75dR931 here we need to pass this as `true`

Reply:
That link is unfortunately broken, but do you mean the host in `executeCommandLine.ts`'s `performIncrementalCompilation` needs to set true?

Reply:
Yeah, found it.

Reply:
If only I could make this option required so we didn't miss any :(